includes/definitions/fiberhome-switch.yaml

```diff
@@ -0,0 +1,19 @@
+os: fiberhome-switch
+text: 'FiberHome Switch'
+type: network
+icon: fiberhome
+discovery:
+    - sysObjectID:
+        - .1.3.6.1.4.1.3807.1.582815
```

Comment:
Maybe just drop these to one and use `.1.3.6.1.4.1.3807.1.` Our stats don't show this would pick up any other devices.

Reply:
noted and done.